### ISSUE TITLE
Typo in Problem Statement - 852. Peak Index in a Mountain Array

### ISSUE THREAD

**Issue by charitra1022** [LeetCode-Feedback/LeetCode-Feedback#8446]
#### Your LeetCode username
charitra1022

#### Category of the bug
- [x] Question
- [x] Typo
- [x] Insufficient info

- [ ] Solution
- [ ] Language
- [ ] Missing Test Cases

#### Description of the bug
Minor syntactical typo in **first line** of problem statement.
Insufficient information in the question statement. Info to add just before **last line** in problem statement.

#### Original problem statement
An array ```a``` mountain if the following properties hold:
....
You must solve it in ```O(log(arr.length))``` time complexity.

#### Corrected statement
An array ```a``` is a mountain array if the following properties hold:
....
In other words, find the index of the **peak** element of the array.
You must solve it in ```O(log(arr.length))``` time complexity.

#### Screenshots
![Screenshot here](https://i.ibb.co/jHYsTGH/852.png)

**Comment by LC-Pam** [comment]
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by LC-Pam** [comment]
Thank you for your time. We will keep the statement.  We appreciate your support!
